server/models/Collective.js

```diff
@@ -271,12 +271,17 @@ export default function (Sequelize, DataTypes) {
         },
       },
 
-      expensePolicy: DataTypes.TEXT, // markdown
+      expensePolicy: {
+        type: DataTypes.TEXT, // markdown
+        validate: {
+          len: [0, 16000], // largest policy is 15,500, all the rest are under 10,000
```

Comment:
Maybe we need more space for safety? Like 32000.

Reply:
Would we want it getting much longer than 16,000 though?